### ISSUE TITLE
why aptos-cli binary built by myself file has 2.3G, while 15M?

### ISSUE THREAD

**Issue by yasyzb** [aptos-labs/aptos-core#6394]
### Please note that this issue tracker is not a help form and this issue will be closed.

Please contact the development team on [Discord](https://discord.gg/aptoslabs)

**Comment by JoshLind** [comment]
Yes, please see the CLI build profile here: https://github.com/aptos-labs/aptos-core/blob/main/Cargo.toml#L503. You should use this when building the CLI. It'll remove debug symbols, strip the binary, etc.

Closing this for now, but feel free to reopen if you still can't produce the correct binary size 😄
